spring-boot-tests/spring-boot-smoke-tests/spring-boot-smoke-test-secure-webflux/src/test/java/smoketest/secure/webflux/SampleSecureWebFluxCustomSecurityTests.java

```diff
@@ -55,10 +55,9 @@ void userDefinedMappingsSecure() {
 	}
 
 	@Test
-	void healthAndInfoDoNotRequireAuthentication() {
+	void healthDoNotRequireAuthentication() {
```

Comment:
I guess this should be `healthDoesNotRequireAuthentication`?

Reply:
yes

Reply:
fixed